app/assets/javascripts/alert.js

```diff
@@ -0,0 +1,28 @@
+(function () {
+  var float_alert;
+  float_alert = typeof exports !== 'undefined' && exports !== null ? exports : this;
```

Comment:
The two lines above can be simplified by replacing it with:
var float_alert = exports || this;